disnake/flags.py

```diff
@@ -909,6 +909,27 @@ def dm_typing(self):
         """
         return 1 << 14
 
+    @flag_value
+    def guild_scheduled_events(self):
+        """:class:`bool`: Whether guild scheduled event related events are enabled.
+
+        This corresponds to the following events:
+
+        - :func:`on_guild_scheduled_event_create`
+        - :func:`on_guild_scheduled_event_update`
+        - :func:`on_guild_scheduled_event_delete`
+        - :func:`on_guild_scheduled_event_subscribe`
+        - :func:`on_guild_scheduled_event_unsubscribe`
```

Comment:
These two also need the `members` intent (at least), should be mentioned in the `Note that due to an implicit relationship ...` block for the `members` intent.